### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by koji** [ml5js/ml5-website-v01#20]
To translate this I need to create ja.json in [this dir](https://github.com/ml5js/ml5js.github.io/tree/source/website/i18n), right?

**Comment by cvalenzuela** [comment]
Yep, but there are also a couple of more steps. I think the best approach is to follow the Docusarus instructions:

https://docusaurus.io/docs/en/translation.html#translating-your-existing-docs

Happy to help you with this!

**Comment by koji** [comment]
@cvalenzuela Thanks.
I created ja.json and translated it. I will ask you some questions on that.
